docs/kiali/01-01-kiali.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Overview
+---
+
+Kyma uses [Kiali](https://www.kiali.io) to enable validation, observe the Istio Service Mesh, and provide details on microservices included in the Service Mesh and connections between them.
+Kiali offers a set of dashboards and graphs that allow to have the full service mesh at a glance and quickly identify problems and configuration issues.
+Some of its features are:
```

Comment:
That list of features is changing and some are even not part of the default setup as for example the Istio wizards or the custom dashboards, or the tracing integration.
I prefer to not list it here but just have the link to the upstream docu

Reply:
Removed list and linked equivalent page in kiali docs.